src/bin/pgcopydb/parsing_utils.c

```diff
@@ -602,7 +602,7 @@ parse_pguri_info_key_vals(const char *pguri,
 			foundDBName = true;
 			uriParameters->dbname = strdup(option->val);
 		}
-		else if (value != NULL && !streq(value, ""))
+		else if (!streq(value, ""))
```

Comment:
I would like a comment that says that given previous tests, value can't be NULL. That's useful in case of a later refactoring, we might appreciate having a shortcut into re-discovering that.

Reply:
The reason for the change is that `streq` already has the null checks by definition.